egs2/must_c/asr1/conf/slurm.conf

```diff
@@ -0,0 +1,17 @@
+# Default configuration                                                                                                                                                        
+command sbatch --export=PATH
+option name=* --job-name $0
+default time=48:00:00
+option time=* --time $0
+option mem=* --mem-per-cpu $0
+option mem=0
+option num_threads=* --cpus-per-task $0
+option num_threads=1 --cpus-per-task 1
+option num_nodes=* --nodes $0
+default gpu=0
+option gpu=0 -p RM-shared
```

Comment:
you can copy the slurm from TEMPLATE to there, otherwise, you may not pass the CI